### ISSUE TITLE
Recommendation ：development tool  edit the code value or HTML (data)?

### ISSUE THREAD

**Issue by pgy866** [OrchardCMS/Orchard#7751]
This development tool is very good, it is recommended to consider whether you can add a function, you can analyze the view (UI) directly edit the code value or HTML (data)?
![3](https://user-images.githubusercontent.com/28694676/27763603-e1be91a0-5eb8-11e7-991e-bfc420fbafb7.png)
.
.
.

![1](https://user-images.githubusercontent.com/28694676/27763602-e1bc49ae-5eb8-11e7-9634-18cacbc1fbb5.png)
.
.
.

![2](https://user-images.githubusercontent.com/28694676/27763601-e1bc3068-5eb8-11e7-901b-11718c5f6ed8.png)

**Comment by sebastienros** [comment]
Are you asking to be able to edit the files directly from the admin?

**Comment by pgy866** [comment]
Yes, orchard development tool is very well, I want to add more easily to the design of creating templates, where some UI shows gif, online experience can be:>>>http://www.kooboo.org/Admin/Pages?SiteId=418531f0-a8ed-2fd4-a3a3-27602d799e98      Username:123456   Password:123456



![5](https://user-images.githubusercontent.com/28694676/28244747-0817bf70-6a26-11e7-908b-a5d99dbcfac3.gif)
![6](https://user-images.githubusercontent.com/28694676/28244748-0943c65a-6a26-11e7-8df4-c0d961086833.gif)

**Comment by sebastienros** [comment]
This can't work in a multi-node environment, hence we can't provide this feature.

**Comment by pgy866** [comment]
@sebastienros 
Then the administrator can directly re-development tool edit tracking some of the data?
